recipes/billingegroup-meta/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "billingegroup-meta" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True  # [py36 or win]
+  number: 0
+
+requirements:
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - seaborn
+    - fabio
+    - pims
+    - yaml
+    - pandas
+    - jupyter
+    - pyFAI  # [linux]
+
+test:
+  imports:
+    - matplotlib
+    - pyFAI
+
+about:
+  home: https://thebillingegroup.com/
+  license: MIT
```

Comment:
It seems like we have many different licences for the underlying packages. Should I put all those in the license slot? Can I have a licence for our metapackage be separate from the packages we are including?